src/css/profile/wearable/changeable/theme-circle/listview.less

```diff
@@ -95,6 +95,31 @@
 			}
 		}
 
+		&.li-fadeout {
+			&-left, &-right {
+				&::after {
+					content: "";
+					position: absolute;
+					width: 34 * @px_base;
+					height: 100%;
+				    background: linear-gradient(90deg, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.52), rgba(0, 0, 0, 1));
+					transform: rotate(-180deg);
+					background-image: linear-gradient(90deg, rgba(0, 0, 0, 0.00) 40%, rgba(0, 0, 0, 0.52) 79%, #000000 100%);
```

Comment:
Should I use the value of a as 0.00?
I think it is good to use it as 0.

Reply:
I copied all values from guide but I think adding double notation is not necessary. Thanks.